at-hoangtran/Week3/Day4/Exercise1-Day4.rb

```diff
@@ -0,0 +1,5 @@
+def repeat(str)
+  arr = str.split(//)
+  arr.uniq{ |x| }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/3cf28bfecc132fb6fcec164677df66b52ed8042f/at-hoangtran/Week3/Day4/Exercise1-Day4.rb#L3"></span>Layout/SpaceBeforeBlockBraces: Space missing to the left of {.</td>
    </tr>
  </tbody>
</table>